.github/workflows/test.yml

```diff
@@ -88,6 +86,7 @@ jobs:
         run: node utils/gen.js && node_modules/.bin/turbo run test --cache-dir=".turbo" --log-order=stream --scope=${{matrix.packageName}} --no-deps -- ${{ join(matrix.testPaths, ' ') }}
         shell: bash
         env:
+          JEST_JUNIT_OUTPUT_FILE: ${{github.workspace}}/.junit-reports/${{matrix.scriptName}}-${{matrix.packageName}}-${{matrix.chunkNumber}}-${{ matrix.runner }}.xml
```

Comment:
Under our current test sharding strategy we technically would only generate a single `junit.xml` file per run/container. But, I wanted to bake in a bit of safety in case that changes in the future and we end up with multiple files per run.